program/CORE-V IDE prelminary project proposal.md

```diff
@@ -0,0 +1,251 @@
+- No funding requirements yet identified
+
+
+## Funding supplied by members - requirement and availability
+- No funding requirements yet identified
+
+
+
+## Architecture diagram
+*Need diagram showing SW architecture and also real-world physical setup for intended use. Action to Alexander* 
+
+## Who would make use of OpenHW output
+CORE-V CDT will be used by developers created code for the following target platforms
+- CV32E40P FPGA platform
+- *expand/detail this list*
+
+## Project license model
+
+This project will be released under EPL 2.0
+* Explain how the code under other licenses, such as GNU PL, will be incorporated *
+
+## Description of initial code contribution, if required
+* Explain initial source for all code contributions, and which initial code contributions will have Contribution Questionnaire generated *
+
+
+## Repository Structure
+* Explain the initial OpenHW repository structure and how it will be migrated to Eclipse Foundation management*.
+
+
+
+
+## Project distribution model
+
+The project code will be distributed using an Eclipse p2 repository and an archive for each target platform.
+
+* To be discussed with Mikael Barbero and described here*
+
+
+## Preliminary Project plan
+
+The following are initial steps forming elements of a project plan. A full project plan is not yet available.
+
+*Note, at Preliminary Project Launch we don't need a full project plan but we should enumerate key steps and timelines where available*
+
+### Steps to First Integration at end of 2020 (not neccesarily in the following order)
+1. Preliminay project proposal (this document)- Duncan Bees/SW TG members, first draft Aug 28 2020 
+2. Project plan
+3. OpenHW Project Approval
+4. Eclipse Project Initiation and initial Contribution Questionnaires
+5. CORE-V artwork for IDE (done) - Rick O'Connor/Alexander Fedorov
+6. CORE-V branding text for IDE (done) - Rick O'Connor
+7. Github repository structure created (done) - Jeremy Bennet /Alexander Fedorov/ Florian Zaruba 
+8. Runnable stub IDE (done) - Alexander Fedorov
+9. Simple hello world application to take from source to binary (not defined as yet, tester of entire toolchain). Could Sebastian from Thales help?
+10. OpenHW Reference tools available - first repository created - binaries available from Embecosm
+11. OpenOCD software (On Chip Debuggger) available from openOCD project, which supports RISC-V, binaries likely will be imported - Hugh O'Keeffe to comment
+12. Hardware debugger connector JTAG<->USB  presumably from Ashling
+13. Integration of toolchain and IDE
+14. Integration of debugger to the IDE
+15. Eclipse download mechanics definition 
+- Assistance expected from Mikael Barbero
+- Define build process using signed content for Java (presumably Eclipse Foundation)
+- Define download area in Eclipse - OpenHW will organise the structure
+- Write the Build and Delivery Process document
+16. Setup kanban board/project backlog mechanism
+

```

Comment:
This is far too much detail and mixes up deliverables, milestones and prerequisites. Each of these should have a section, but for a problem of this scale they will be very simple.